### DISCUSSION TITLE
Not dowloading/using the correct Chromium binary for snapshots on macOS(M1)

### DISCUSSION THREAD

**Discussion by prsdta** [percy/cli#586] [Support] [answered]
Despite the docs saying the CLI will download and use its own version of Chromium, I keep running onto an issue on my machine:

```
Selenium::WebDriver::Error::SessionNotCreatedError:
   session not created: This version of ChromeDriver only supports Chrome version 92
   Current browser version is 94.0.4606.81 with binary path /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
```

When running with debug flag I also get sometimes:
```
[percy] Skipping visual tests - Failed to launch browser. Timed out after 30000ms
```

Setting manually the `executable` option in the percy config to the executable I had in the `.local-chromium` folder didn't help either.
When installing the package or running the percy excutable, I do not see any installation step (though it seems to populate the `.local-chromium` folder).

Using Yarn, Capybara w/ Selenium (percy-capybara 5.0.0), `@percy/cli 1.0.0-beta.67` (but same behavior on latest), macOS Big Sur on a M1 machine.

**Comment by Robdel12** [comment, marked as answer]
Hey @prsdta! This error is from your test suite and unrelated to Percy:

```
Selenium::WebDriver::Error::SessionNotCreatedError:
   session not created: This version of ChromeDriver only supports Chrome version 92
   Current browser version is 94.0.4606.81 with binary path /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
```

You'll need to upgrade the version of `ChromeDriver` to match the version installed on your machine. You can use a gem like `webdrivers` to manage that: https://github.com/titusfortner/webdrivers#chromechromium

You'll want to undo any of the `executable` config set in the `.percy.yml`. Percy's asset discovery browser is entirely separate from the test suites browser.

**Reply by prsdta** [reply]
My bad, I had the assumption the two were linked and that didn't help me debug my issue. I'll look into the selenium side. 
Thanks for the quick answer.
